experiment/run_experiment.py

```diff
@@ -127,26 +127,32 @@ def validate_fuzzer(fuzzer: str):
 
 def validate_fuzzer_config(fuzzer_config):
```

Comment:
We should probably validate that no 2 variants have the same name (either variants of same fuzzer or different ones).

Reply:
Added a check while we're parsing the variant configs.